Mathlib/Algebra/Ring/Defs.lean

```diff
@@ -260,6 +260,8 @@ multiplication by zero law (`MulZeroClass`). -/
 class NonUnitalCommSemiring (α : Type u) extends NonUnitalSemiring α, CommSemigroup α
 #align non_unital_comm_semiring NonUnitalCommSemiring
 
+/-- A commutative semiring (`CommSemiring`) is a semiring (`Semiring`)
```

Comment:
If you're trying to follow the existing docstrings, it would rather be
```suggestion
/-- A commutative semiring is a semiring (`Semiring`)
```
Honestly I see no point in repeating information so clearly accessible from the `extends` clause, so I would rather do
```suggestion
/-- A commutative semiring is a semiring with commutative multiplication. -/
```

Reply:
Yes, I agree. Committed.